astroid/tests/unittest_brain_numpy.py

```diff
@@ -0,0 +1,246 @@
+#-*- encoding=utf-8 -*-
+# Copyright (c) 2013-2014 Google, Inc.
```

Comment:
If you wrote this yourself and it doesn't contain code from other files, feel free to just add yourself here!